libr/core/cconfig.c

```diff
@@ -1491,7 +1491,7 @@ static bool cb_dirsrc(void *user, void *data) {
 static bool cb_cfgsanbox_grain(void *user, void *data) {
 	RConfigNode *node = (RConfigNode*) data;
 	if (strstr (node->value, "?")) {
-		eprintf ("Usage: comma separated grain types to be masked out by the sandbox.\n");
+		eprintf ("comma separated grain types to be masked out by the sandbox.\n");
```

Comment:
User is an instance of core. Use rcorehelp here

Reply:
Using R_LOG_INFO can be also an option in some situations but for help its better to be consistent if possible